common/aws/ecs/ecs.go

```diff
@@ -392,6 +392,12 @@ func (this *ECS) RunTask(cluster, taskDefinition string, count int64, startedBy
 		Count:          aws.Int64(count),
 		StartedBy:      startedBy,
 		Overrides:      taskOverride,
+		PlacementStrategy: []*ecs.PlacementStrategy{
```

Comment:
no, I thought it was something you put in a task definition, but it turns out is only a param on these aws calls.